NEXT.md

```diff
@@ -0,0 +1,112 @@
+This branch contains the code for a "next version" of Vitessce which will have the following key features:
+- Coordinated multiple views
+- Multi-dataset support
+- Global state support
+
+Please see the changelog for detailed information about what has been changed so far.
+
+## Coordination object model
+
+The implementation of coordinated multiple views here follows the coordination object model proposed by Boukhelifa and Rodgers (Information Visualization 2003).
+
+Below left is Figure 1 from Boukhelifa and Rodgers. Below right is that figure adapted to the Vitessce view config. ([view the diagram on google drawings here](https://docs.google.com/drawings/d/1jsNd2aG3OFlHfNzI3nfOl6UpMACw9JKyexCQUEd31fc/edit)).
+
+![Coordination object model in vitessce](https://user-images.githubusercontent.com/7525285/89790691-49008b00-daf0-11ea-95b5-cd74fe3499af.png)
```

Comment:
```suggestion
Below left is Figure 1 from Boukhelifa and Rodgers. Below right is that figure adapted to the Vitessce view config. 

[![Coordination object model in vitessce](https://user-images.githubusercontent.com/7525285/89790691-49008b00-daf0-11ea-95b5-cd74fe3499af.png)](https://docs.google.com/drawings/d/1jsNd2aG3OFlHfNzI3nfOl6UpMACw9JKyexCQUEd31fc/edit)
```
So clicking on it takes you to the google doc... but you might feel this is less clear / more confusing. Either way.

Reply:
About the diagram: Does "A" in each of the coordinations refer to the same thing? I think that's not the case... each is its own namespace. 
- Perhaps use IDs with semantics for the example?
- Perhaps add lines to the diagram to make it clear what each "A" (or other ID) links?

Reply:
Diagram nitpick: I think using the same corner radius on all the round-rects will look tidier.